lnbits/wallets/fake.py

```diff
@@ -42,44 +51,55 @@ async def create_invoice(
         memo: Optional[str] = None,
         description_hash: Optional[bytes] = None,
         unhashed_description: Optional[bytes] = None,
-        **kwargs,
+        expiry: Optional[int] = None,
+        payment_secret: Optional[bytes] = None,
+        **_,
     ) -> InvoiceResponse:
-        data: Dict = {
-            "out": False,
-            "amount": amount * 1000,
-            "currency": "bc",
-            "privkey": self.privkey,
-            "memo": memo,
-            "description_hash": b"",
-            "description": "",
-            "fallback": None,
-            "expires": kwargs.get("expiry"),
-            "timestamp": datetime.now().timestamp(),
-            "route": None,
-            "tags_set": [],
-        }
+        tags = Tags()
+
         if description_hash:
-            data["tags_set"] = ["h"]
-            data["description_hash"] = description_hash
+            tags.add(TagChar.description_hash, description_hash.hex())
         elif unhashed_description:
-            data["tags_set"] = ["d"]
-            data["description_hash"] = hashlib.sha256(unhashed_description).digest()
+            tags.add(
+                TagChar.description_hash,
+                hashlib.sha256(unhashed_description).hexdigest(),
+            )
         else:
-            data["tags_set"] = ["d"]
-            data["memo"] = memo
-            data["description"] = memo
-        randomHash = (
+            tags.add(TagChar.description, memo or "")
```

Comment:
- does `Tags` take care to set both the `memo` and `description` fields?

Reply:
there is no memo field in bolt11